dbt_project.yml

```diff
@@ -34,7 +34,7 @@ vars:
     snowplow__yauaa_context: "{{ source('atomic','nl_basjes_yauaa_context_1') }}"
     snowplow__sessions_table: "{{ ref('snowplow_web_sessions') }}" # Change to your custom sessions table if you have disabled the standard sessions table in favour of a custom version. Advanced config.
     # Variables - Standard Config
-    snowplow__start_date: '2020-01-01'
+    snowplow__start_date: '2023-01-01'
```

Comment:
Assume this was a local change you forgot to remove before committing, we all do it 😅